integration/testdata/TestMavenIntegration/cloud-sdk-spring-archetype/.gitignore

```diff
@@ -0,0 +1,19 @@
+**/credentials.*
```

Comment:
Is this file still necessary, once the issue with the temp-folder is fixed?

Reply:
this is just part of the archetype, and I won't remove the gitignore file for reasons. what if you manually do a mvn package in there, would you want to git add `target` then?